__fixtures__/unit/helper.js

```diff
@@ -1,5 +1,6 @@
 const _ = require('lodash')
 const yaml = require('js-yaml')
+const moment = require('moment-timezone')
```

Comment:
I believe moment is no longer being maintained. We may want to look into using Luxon or another alternative moving forward. I do understand that we've had this as a dependency in the code base -- something we may want to address in the future.

Reply:
created a issue to track this, #525